pkgdb2client.py

```diff
@@ -0,0 +1,25 @@
+import pkgdb2client
+import json
+
+def get_packages(name):
```

Comment:
At the same time I would rename the method to getPackageDistros are something similar that resembles more the method's purpose. Or maybe getPackageInfo from which one can construct a list of branches for a package name.